components/typography/typography.module.ts

```diff
@@ -21,7 +20,7 @@ import { NzTextEditComponent } from './text-edit.component';
 import { NzTypographyComponent } from './typography.component';
 
 @NgModule({
-  imports: [CommonModule, NzIconModule, NzToolTipModule, NzInputModule, NzI18nModule, NzTransButtonModule, NzCopyToClipboardServiceModule],
+  imports: [CommonModule, NzIconModule, NzToolTipModule, NzInputModule, NzI18nModule, NzTransButtonModule],
```

Comment:
missing ClipboardModule

Reply:
Clipboard (as a service) is provided in root. It's unnecessary to import `ClipboardModule` if we don't use `[cdkCopyToClipboard]`.

Reply:
https://github.com/angular/components/blob/e311791bcaec9ca2a850def19e1a8b954dea4bb1/src/cdk/clipboard/clipboard.ts#L14-L23

Reply:
yes, but we don't know what the code will happen in the future, it is safe to import the module when using the service inside it